Terminal.Gui/Views/TextField.cs

```diff
@@ -41,7 +41,7 @@ public class TextField : View {
 		/// <remarks>
 		///   The passed <see cref="EventArgs"/> is a <see cref="ustring"/> containing the old value. 
 		/// </remarks>
-		public event EventHandler<ustring> Changed;
+		public Action<ustring> Changed;
```

Comment:
`TextChanged` (to be consistent with `TextView`)?

Reply:
But wouldn't that break with code already made by users?

Reply:
This already does, right?

Reply:
No.

Reply:
But if it is to break it will have to be now because we should not be constantly changing the names of public variables constantly. It is necessary to have the prior approval of @migueldeicaza so we don't move back and forth.